lib/nexpose/scan_template.rb

```diff
@@ -577,5 +577,39 @@ def aces_level
     def aces_enabled?
       return 'full' == aces_level
     end
+
+    # Enable or disable the debug logging.
+    # @param [Boolean] enable Enable or disable the debug logging.
+    def enable_debug_logging=(enable)
```

Comment:
There are methods with the same prefix serving this purpose in the file. Don't see a big difference to make the change.

```
$ grep -B2 "def enable_.*=(enable)" lib/nexpose/scan_template.rb
    # Enable/disable IP stack fingerprinting
    # @param [Boolean] enable Enable or disable IP stack fingerprinting
    def enable_ip_stack_fingerprinting=(enable)
--
    # Enable/disable ICMP device discovery
    # @param [Boolean] enable Enable or disable ICMP device discovery
    def enable_icmp_device_discovery=(enable)
--
    # Enable/disable TCP device discovery
    # @param [Boolean] enable Enable or disable TCP device discovery
    def enable_tcp_device_discovery=(enable)
--
    # Enable/disable UDP device discovery
    # @param [Boolean] enable Enable or disable UDP device discovery
    def enable_udp_device_discovery=(enable)
--
    # Enable or disable TCP port scanning.
    # @param [Boolean] enable Enable or disable TCP ports
    def enable_tcp_service_discovery=(enable)
--
    # Enable or disable UDP service discovery
    # @param [Boolean] enable Enable or disable UDP service discovery
    def enable_udp_service_discovery=(enable)
--
    # Enable or disable the debug logging.
    # @param [Boolean] enable Enable or disable the debug logging.
    def enable_debug_logging=(enable)
--
    # Enable or disable the enhanced logging.
    # @param [Boolean] enable Enable or disable the enhanced logging.
    def enable_enhanced_logging=(enable)
```